include/flash_map_pm.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef FLASH_MAP_PM_H_
+#define FLASH_MAP_PM_H_
+
+#include <pm_config.h>
+#include <sys/util.h>
+
+/* Aliases for zephyr - mcuboot/ncs style naming */
+#define image_0 mcuboot_primary
+#define image_1 mcuboot_secondary
+
+#if (CONFIG_SETTINGS_FCB || CONFIG_SETTINGS_NVS)
+#define storage settings_storage
```

Comment:
I don't understand the scope here, but this approach looks a bit sketchy to me. The token "storage" seems like a pretty likely variable name or something that will be used in other contexts. Is there really no other way?

Reply:
This is sketchy, but I don't see how it could be done without either:
- Downstream patches wherever 'storage' is used
or 
- Upstream change to use symbol instead of `storage` which is deduced from `flash_map.h` so that we can provide our own variant.

Reply:
Isn't this the name of a partition? I thought that was the namespace that PM and a fixed-partitions DT approach shared.

Reply:
This is the name of the partition, but unfortunately with in the PM definitions there is not a one to one mapping with the upstream partition names. In the PM defines we separate the `storage` used between different subsystems (`littlefs_storage`, `settings_storage`, `nvs_storage`), since in most cases these can not live together using the same partition.

Reply:
Should we patch these upstream then, to create either additional partitions or some other way to alias these? I would really like to avoid tricks like this if we can.

Reply:
Yes, I have discussed this with @de-nordic , and will come up with a proposal some time soon.